pyproject.toml

```diff
@@ -0,0 +1,33 @@
+[build-system]
+requires = [
+    "setuptools>=42",
+    "wheel",
+    "setuptools_scm[toml]>=3.4"
+]
+build-backend = "setuptools.build_meta"
+
+[tool.setuptools_scm]
+local_scheme = "no-local-version"
+write_to = "gdcdatamodel/_version.py"
+
+[tool.black]
+line-length = 88
+target-version = [
+    'py37',
+]
+include = '\.pyi?$'
+
+[tool.coverage.run]
+branch = true
+context = "unit tests"
+source = ["gdcdatamodel"]
+
+[tool.coverage.html]
+title = "gdcdatamodel coverage report"
+directory = "htmlcov"
+show_contexts = true
+
+[tool.isort]
+profile = "black"
+known_third_party = ["addict", "authutils", "cdiserrors", "cdislogging", "ciso8601", "deepdiff", "defusedcsv", "dotenv", "envelopes", "flask", "flask_cors", "gdc_ng_models", "gdcdictionary", "httmock", "indexclient", "indexd", "indexd_test_utils", "jsonschema", "mock", "pkg_resources", "psqlgraph", "pytest", "requests", "setuptools", "simplejson", "six", "sqlalchemy", "yaml"]
```

Comment:
What's this for? Is it needed?
Just wondering about keeping these updated.

Reply:
for isort to know how to automatically sort imports. 
Even thought we do not have isort for this repo yet. 
I think isort was removed from pre-commit-config for some release.